cogboard-webapp/src/components/widgets/index.js

```diff
@@ -55,7 +55,7 @@ const widgetTypes = {
     dialogFields: ['TimeZoneId', 'DateFormat', 'TimeFormat', 'DisplayDate', 'DisplayTime', 'DateTimeSize']
   },
   IFrameEmbedWidget: {
-    name: 'IFrame Embed widget',
+    name: 'Iframe Embed widget',
```

Comment:
Rename should be also applied to files names for better readability. I already done this on separate branch. You can safely delete the change from this branch 😀

Reply:
done :)